main/src/library/Fixpoint/Interpreter.flix

```diff
@@ -14,19 +14,19 @@
  * limitations under the License.
  */
 
-mod Fixpoint {
+mod Fixpoint.Interpreter {
 
     use Fixpoint.Ast.Ram.{RamStmt, RelOp, RamSym, RamTerm, BoolExp, RowVar, toDenotation}
     use Fixpoint.Ast.Denotation
 
-    type alias Database[v: Type, r: Region] = MutMap[RamSym[v], MutMap[Vector[v], v, r], r]
-    type alias SearchEnv[v: Type, r: Region] = (Array[Vector[v], r], Array[v, r])
+    pub type alias Database[v: Type, r: Region] = MutMap[RamSym[v], MutMap[Vector[v], v, r], r]
+    pub type alias SearchEnv[v: Type, r: Region] = (Array[Vector[v], r], Array[v, r])
 
-    def interpret(rc: Region[r], stmt: RamStmt[v]): Database[v, r] \ r with Order[v] =
+    pub def interpret(rc: Region[r], stmt: RamStmt[v]): Database[v, r] \ r with Order[v] =
         interpretWithDatabase(rc, MutMap.new(rc), stmt)
 
-    def interpretWithDatabase(rc: Region[r], db: Database[v, r], stmt: RamStmt[v]): Database[v, r] \ r with Order[v] =
-        masked_cast(notifyPreInterpret(stmt));
+    pub def interpretWithDatabase(rc: Region[r], db: Database[v, r], stmt: RamStmt[v]): Database[v, r] \ r with Order[v] =
```

Comment:
For example, this may need to be marked `@Internal`

Reply:
I will do, I'm making a jar now...

Reply:
It seems we did not manage on master either: https://api.flix.dev/Fixpoint.html

Reply:
It is minor point, in any case.